### ISSUE TITLE
Priority of Safe Links & Safe attachment policies

### ISSUE THREAD

**Issue by ShawnHag** [MicrosoftDocs/microsoft-365-docs#8766]
I was wondering how few policies for a same user will take effect.

In the section about the priority of policies, it stated that policies are processed in priority order.
I have also checked the following link which was given in the description 
https://docs.microsoft.com/microsoft-365/security/office-365-security/how-policies-and-protections-are-combined
(but wasn't quiet sure which category Safe links and safe attachments belonged to in the table )

I understand that policies are processed in order from higher priority policies,
but what will happen in the following scenarios?
< Lower Priority policy having a weaker action >
Policy A for Safe attachment: Priority 0, Block mode
Policy B for Safe attachment: Priority 1, Monitor mode

After an e-mail with an attachment is blocked with policy A, 
since policies are being processed in order, will Policy B process the monitor action to an e-mail that was just blocked by policy A and let it pass through?

< Lower Priority policy having a stronger action >
Policy A for Safe attachment: Priority 0, Monitor mode
Policy B for Safe attachment: Priority 1, Block mode

Not very sure if all policies will be processed every single time.

Thank you in advance,
Shawn
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1541da6f-af9a-05d5-fa34-b1a5df61424f
* Version Independent ID: 88e111ed-67f7-41bc-d823-863ee7ea2a2e
* Content: [Set up Safe Links policies in Microsoft Defender for Office 365 - Office 365](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/set-up-safe-links-policies?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/set-up-safe-links-policies.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/set-up-safe-links-policies.md)
* Product: **m365-security**
* Technology: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@ShawnHag, I'm looking for updates to the 'order of protection' table in that other topic per [this issue](https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8514).

The thing to remember is: a given protection policy (spam, phishing, Safe Links, Safe Attachments, etc.) is only applied to a message ONCE, and which policy is applied depends on the priority. There is NEVER multi-processing of a message by (for example) Safe Attachments.

If I understand your questions correctly, when Policy A is set to Block, then the message is blocked. But if Policy A is set to Monitor, then Safe Attachments allows the message to pass through. Policy B never affects the message. A different type of protection and its policies that's applied to the messages AFTER Safe Attachments could still catch the message (impersonation or spoofing protection in anti-phishing policies, for example).

**Comment by chrisda** [comment]
The big clarification I should add for my previous statements is: when the same recipient is defined in multiple policies (i.e., multiple policies of the same type apply to the same recipient). The message is evaluated for every recipient in the policy, but policy processing stops after the first policy hit, which is a good reason not to assign people to multiple policies of the same type.

**Comment by ShawnHag** [comment]
@chrisda 
Hello! 

Thank you so much for the clarification.
My understanding is very clear now.

I now understand that only one policy (when theres few policies for one type like Safe attachments ) will take in effect to an e-mail. But theres a possibility that a different type of policy ( other than Safe attachments in this case) could effect the e-mail after.

Once again, thank you!

**Comment by yogkumgit** [comment]
@ShawnHag If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
